docker-compose-dev.yaml

```diff
@@ -268,6 +270,65 @@ services:
     devices:
       - "/dev/fuse"
 
+  badgerdoc-elasticsearch:
+    container_name: badgerdoc-elasticsearch
+    image: amazon/opendistro-for-elasticsearch:latest
+    environment:
+      - node.name=es01
+      - cluster.name=es-docker-cluster
+      - cluster.initial_master_nodes=es01
+      - bootstrap.memory_lock=true
+      - opendistro_security.disabled=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+    healthcheck:
+      test: curl --fail http://localhost:9200 || exit 1
+      interval: 30s
+      timeout: 3s
+      retries: 10
+      start_period: 30s
+    networks:
+      - badgerdoc
+    ports:
+      - "9204:9200"
+    volumes:
+      - elasticdata:/usr/share/elasticsearch/data
+
+  badgerdoc-embeddings:
+    image: badgerdoc_embeddings
+    container_name: embeddings
+    environment:
+      - ROOT_PATH=/search/embeddings
+    networks:
+      - badgerdoc
+    ports:
+      - "0.0.0.0:3334:3334"
+
+  badgerdoc-search:
+    image: badgerdoc_search
+    container_name: search
+    env_file:
+      - .env
+    environment:
+      - ROOT_PATH=/search
+    working_dir: /opt/search
+    ports:
+      - 8084:8080
```

Comment:
All microservices proxies from `reproxy` to keep the same port for all microservices. You need to remove ports bindings here and add `EXPOSE` to dockerfile